webapp/src/components/rhs/rhs_new_tab.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {useDispatch} from 'react-redux';
+import styled from 'styled-components';
+
+import {
+    FormatListNumberedIcon,
+    LightbulbOutlineIcon,
+    PlaylistCheckIcon,
+} from '@mattermost/compass-icons/components';
+
+import {createPostImmediately} from 'mattermost-redux/actions/posts';
+
+import RHSImage from '../assets/rhs_image';
+
+import {Button} from './common';
+
+const CreatePost = styled((window as any).Components.CreatePost)`
```

Comment:
Again this will break if the component is not available.

Reply:
Fixed.